src/modules/powerrename/PowerRenameContextMenu/AppxManifest.xml

```diff
@@ -11,9 +11,9 @@
   xmlns:uap10="http://schemas.microsoft.com/appx/manifest/uap/windows10/10"
   xmlns:com="http://schemas.microsoft.com/appx/manifest/com/windows10"
   IgnorableNamespaces="uap uap2 uap3 rescap desktop desktop4 desktop5 uap10 com">
-  <Identity Name="PowerRenameContextMenu" ProcessorArchitecture="neutral" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" Version="1.0.0.0" />
+  <Identity Name="Microsoft.PowerToys.PowerRenameContextMenu" ProcessorArchitecture="neutral" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" Version="1.0.0.0" />
   <Properties>
-    <DisplayName>PowerRenameContextMenu</DisplayName>
+    <DisplayName>Microsoft.PowerToys.PowerRenameContextMenu</DisplayName>
```

Comment:
Why not "PowerToys (Preview) x64 PowerRename Context Menu" as display name? (And same for ImageResizer.)

Reply:
Agreed. Having PowerToys in the Display name would make sense as well.

Reply:
Nevermind, this change does add PowerToys. Wouldn't put x64 since that would mean keeping two different manifests without the need, but having something like "PowerToys PowerRename Context Menu" sounds like a good idea. Other entries seem to have a Display name more like this (looking at "Power Automate, for example)

Reply:
<img width="978" alt="image" src="https://user-images.githubusercontent.com/57057282/192749048-a4832d90-050d-472e-8302-31d502809c13.png">

this is how it looks now